### PR TITLE
Fixes/464 forms without property

### PR DESCRIPTION
**Pull request by sadiqkhoja**
Closes getodk/central#464

#### What has been done to verify that this works as intended?

Integration tests and manual verification

#### Why is this the best possible solution? Were any other approaches considered?

NA

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

None

#### Does this change require updates to user documentation? If so, please file an issue [here](https://github.com/getodk/docs/issues/new) and include the link below.

NA

#### Before submitting this PR, please make sure you have:

- [x] run `npm run test` and `npm run lint` and confirmed all checks still pass OR confirm CircleCI build passes
- [x] verified that any code or assets from external sources are properly credited in comments or that everything is internally sourced

### CODE REVIEW

File: src/components/dataset/summary/row.vue [hunk 1]
```diff
@@ -40,6 +40,7 @@ except according to the terms contained in the LICENSE file.
         <span class="sr-only">&nbsp;{{ $t('addedByThisDraft') }}</span>
         <template v-if="index < inFormProperties.length - 1">{{ $t('common.punctuations.comma') }}<sentence-separator/></template>
       </span>
+      <span v-if="inFormProperties.length === 0">no property to show</span>
```
Comment (sadiqkhoja) [inline src/components/dataset/summary/row.vue:43]: text needs to be finalized

Comment (lognaturel) [reply src/components/dataset/summary/row.vue:43]: Latest proposal is "This form only updates the Entity label". It's specific and helpful. We can reconsider what to do about system properties once we have more than one.

**Comment by matthew-white** [comment]
I took a look at getodk/central-backend#960, so I'd be happy to look at this once it's ready.

File: src/components/dataset/summary/row.vue [hunk 2]
```diff
@@ -40,6 +40,7 @@ except according to the terms contained in the LICENSE file.
         <span class="sr-only">&nbsp;{{ $t('addedByThisDraft') }}</span>
         <template v-if="index < inFormProperties.length - 1">{{ $t('common.punctuations.comma') }}<sentence-separator/></template>
       </span>
+      <span v-if="inFormProperties.length === 0" class="no-properties">{{ $t('noProperties') }}</span>
```
Comment (matthew-white) [inline src/components/dataset/summary/row.vue:43]: How about adding a test of this case?

**Review by matthew-white — COMMENTED** [review]
Does this PR also resolve getodk/central#463? The change to `DatasetSummaryRow` makes me think that's the goal. If so, I think there's one more place to make a change. Or that could come in a follow-up PR.

File: src/components/dataset/overview/connection-to-forms.vue
[the remark this answers is not in the document]
Comment (sadiqkhoja) [reply src/components/dataset/overview/connection-to-forms.vue:1]: right

File: test/components/dataset-summary/row.spec.js [hunk 3]
```diff
@@ -44,4 +44,25 @@ describe('Dataset summary row', () => {
...
+
+  it('should show help text when there is no properties', async () => {
+    const dataset = testData.formDraftDatasetDiffs.createPast(1, { isNew: true, properties: [] }).last();
+    const component = mount(Row, {
+      props: { dataset, projectId: 1 },
+      container: {
+        router: mockRouter('/')
+      }
+    });
+    component.getComponent(RouterLinkStub).props().to.should.be.equal(`/projects/1/datasets/${dataset.name}`);
```
Comment (matthew-white) [inline test/components/dataset-summary/row.spec.js:56]: I think this test will fail once merged to `master`, now that #841 is merged. You could try rebasing, or merge `master` into this branch?

**Review by matthew-white — APPROVED** [review]

**Pull request merged**